docs/source/content/deep_learning/multilayer.rst

```diff
@@ -0,0 +1,106 @@
+##########
+Multi-layer Perceptron
+##########
+
+.. contents::
+  :local:
+  :depth: 3
+
+********
+Overview
+********
+
+A multilayer perceptron (MLP) is a deep, artificial *neural network*. 
+A *neural network* is comprised of layers of nodes which activate at various levels depending on the previous layer's nodes. 
+When thinking about neural networks, it may be helpful to isolate your thinking to a single node in the network.
+
+Multilayer perceptron refers to a neural network with at least three layers of nodes, an input layer, some 
+number of intermediate layers, and an output layer. 
+Each node in a given layer is connected to every node in the adjacent layers. 
```

Comment:
given layer _,_ is connected
I think a comma is needed there

Reply:
I don't think it is, so I left it as is.